src/chainlit/server.py

```diff
@@ -160,17 +162,17 @@ def get_html_template():
     <meta property="og:image" content="https://chainlit-cloud.s3.eu-west-3.amazonaws.com/logo/chainlit_banner.png">
     <meta property="og:url" content="{url}">"""
 
+    js = f"""<script></script>"""
+
```

Comment:
(not needed anymore I guess)

Reply:
I suspect this will be useful for other features, such as theme customisation. For that feature we want to inject JS variables (which are not sensitive) so that the theme has the correct colors on the first render (instead of doing an additional http call to get the colors)